com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -494,6 +494,12 @@ private void SpawnNetworkObjectLocallyCommon(NetworkObject networkObject, ulong
                 {
                     if (playerObject)
                     {
+                        // If there was an already existing player object for this player, then mark it as no longer
```

Comment:
I think for clarity, since `(NetworkManager.ConnectedClients[ownerClientId.Value].PlayerObject` now shows up 3 times in as many lines, I'd hoist this as a local variable